openhexa/sdk/pipelines/pipeline.py

```diff
@@ -21,23 +21,28 @@
 from .utils import get_local_workspace_config
 
 logger = getLogger(__name__)
+PIPELINE_MAX_TIMEOUT = 43200
 
 
 class PipelineConfigError(Exception):
     pass
 
 
-def pipeline(code: str, *, name: str = None) -> typing.Callable[[typing.Callable[..., typing.Any]], "Pipeline"]:
+def pipeline(
+    code: str, *, name: str = None, timeout: int = PIPELINE_MAX_TIMEOUT
```

Comment:
I would keep a default of `None` for the timeout and let the backend set the default value.